web/api/v1/api.go

```diff
@@ -909,11 +911,13 @@ type alertingRule struct {
 }
 
 type recordingRule struct {
-	Name      string           `json:"name"`
-	Query     string           `json:"query"`
-	Labels    labels.Labels    `json:"labels,omitempty"`
-	Health    rules.RuleHealth `json:"health"`
-	LastError string           `json:"lastError,omitempty"`
+	Name           string           `json:"name"`
+	Query          string           `json:"query"`
+	Labels         labels.Labels    `json:"labels,omitempty"`
+	Health         rules.RuleHealth `json:"health"`
+	LastError      string           `json:"lastError"`
```

Comment:
You removed the `omitempty` here, but not on the `alertingRule`. Currently if I have any alerting rules configured, the page crashes because it's trying to access this field, but it doesn't exist.

Might be actually best to keep the `omitempty` in both cases, but make the field optional in the TypeScript interface and check for its existence.

Reply:
Done.